README.md

```diff
@@ -1,144 +1,211 @@
-## This branch is the alpha release of Terminus 1.0.
-> Terminus' master branch now represents our Terminus 1.0 work in progress. For the current 0.x version, check out the 0.13.x branch: [https://github.com/pantheon-systems/terminus/tree/0.13.x](https://github.com/pantheon-systems/terminus/tree/0.13.x)
-
-> Documentation for the alpha release of Terminus 1.0 can be found [here](https://terminusma-static-docs.pantheonsite.io/docs/). For the current 0.x version, visit [https://pantheon.io/docs](https://pantheon.io/docs).
-
-
-Terminus
-========
-
-[![Build Status](https://travis-ci.org/pantheon-systems/terminus.svg?branch=master)](https://travis-ci.org/pantheon-systems/terminus) [![Dependency Status](https://gemnasium.com/pantheon-systems/terminus.svg)](https://gemnasium.com/pantheon-systems/terminus)
+# terminus : Pantheon's Command-Line Interface
+ 
+## Status
+[![Build Status](https://travis-ci.org/pantheon-systems/terminus.svg?branch=master)](https://travis-ci.org/pantheon-systems/terminus)
+[![Dependency Status](https://gemnasium.com/pantheon-systems/terminus.svg)](https://gemnasium.com/pantheon-systems/terminus)
 [![Coverage Status](https://coveralls.io/repos/github/pantheon-systems/terminus/badge.svg?branch=master)](https://coveralls.io/github/pantheon-systems/terminus?branch=master)
-
-Terminus is Pantheon's Command Line Interface (CLI), providing equivalent functionality to the Pantheon Web Dashboard and easier scripting.
+ 
+## About
+Terminus is Pantheon's Command Line Interface (CLI), providing at least equivalent functionality to the Pantheon's
+browser-based Dashboard and easier scripting.
 
 If you would like to contribute, pull requests are welcome!
-
-
-Installation
-------------
-
-**Requirements:**
+ 
+## Dependencies
+### Required
+- A command-line client
 - PHP version 5.5.9 or later
 - [PHP-CLI](http://www.php-cli.com/)
 - [PHP-CURL](http://php.net/manual/en/curl.setup.php)
-- [Composer](https://getcomposer.org/doc/00-intro.md)
 
 Once you have at least the requirements installed, you can install Terminus via Composer or Git. Additionally, you may want to install the optional software below to enhance your use of Terminus:
 
-**Optional but recommended:**
+### Recommended
+- [Composer](https://getcomposer.org/doc/00-intro.md)
 - [Drush](http://docs.drush.org/en/master/install/) (Useful to run incompatible-with-Terminus Drush commands)
 - [WP-CLI](http://wp-cli.org/) (Useful to run incompatible-with-Terminus WP-CLI commands)
 - [Git](https://help.github.com/articles/set-up-git/)
 
-> You can install Terminus just about anywhere on your system. In this README, we'll use `/install/location` to stand in for your chosen installation location.
+You can install Terminus just about anywhere on your system. In this README, we'll use `/install/location` to stand in for your chosen installation location.
 
+## Installation
+### Installing via the Terminus installer
```

Comment:
Merging this effectively releases the installer. I'd rather wait on this until we close out it's open items remaining.